RELEASE.md

```diff
@@ -74,9 +74,9 @@ Please make sure you have these tools installed before starting this process:
 
     -   If you find any critical path that might have been affected, make sure you add/edit the tests in our [automation test suite](https://bitbucket.org/qamine/qa-automation-tests/).
 
-    -   Run the regression tests with our [automation test suite](https://bitbucket.org/qamine/qa-automation-tests/src/8aa6640db54f8cf3ac4f07b70647d66a0ec49739/docs/getting-started.md#run-the-tests). 
+    -   Run the regression tests with our [automation test suite](https://bitbucket.org/qamine/qa-automation-tests/src/8aa6640db54f8cf3ac4f07b70647d66a0ec49739/docs/getting-started.md#run-the-tests).
```

Comment:
Correct the ID to link to the appropriate section. Also, can we link to the version in `master` instead?

```suggestion
    -   Run the regression tests with our [automation test suite](https://bitbucket.org/qamine/qa-automation-tests/src/master/docs/getting-started.md#markdown-header-run-the-tests).
```